.travis.yml

```diff
@@ -14,14 +14,17 @@ before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - export DEPLOY_DOMAIN=https://pr-${TRAVIS_PULL_REQUEST}-evalai.surge.sh
-  - sh -e /etc/init.d/xvfb start
+  - /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16
```

Comment:
I am not sure about this change. @sanketbansal can you please let us know what this line will do?

Reply:
basically, on privacy policy page we do not show **nav bar** for **small screen size** due to which nav bar is not getting **rendered** in headless chrome environment causing values coming to be **null**. So used this line to change screen size.